.ado/azure-pipelines.yml

```diff
@@ -111,3 +111,41 @@ jobs:
       - template: templates/apple-yarn-build.yml
         parameters:
           relative_directory: $(relative_directory)
+
+  - job: Test Build Static Libs
+    displayName: NuGet Publish
+    pool:
+      vmImage: 'macos-10.15'
+      demands: ['xcode', 'sh', 'npm']
+    timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+    cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+
+    steps:
+      - checkout: self
+        persistCredentials: true
+
+      - template: templates/setup-repo-min-build.yml
+
+      # Clean Derived Data
+      - script: |
+          rm -rf $(Build.Repository.LocalPath)/DerivedData
+        displayName: 'Clean DerivedData'
+
+      - script: |
+          sudo gem install cocoapods
+        displayName: 'Install CocoaPods Environment'
+
+      - script: |
+          yarn bundle ios
+        workingDirectory: $(Build.Repository.LocalPath)/apps/ios
+        displayName: 'yarn bundle iOS'
+
+      - script: |
+          yarn bundle macos
+        workingDirectory: $(Build.Repository.LocalPath)/apps/macos
+        displayName: 'yarn bundle macOS'
+
+      # Select proper Xcode version
+      - template: templates/apple-xcode-select.yml
+
+      - template: templates/apple-xcode-build-static-libs.yml
```

Comment:
This is copied from our publish CI pipeline temporarily to build the static libs in our PR CI pipeline and make sure the steps pass.

Testing CI is annoying and the easiest way I've found to do it so far is to do temporary stuff like this that I revert before check in.